ql/src/test/queries/clientnegative/dyn_part_empty.q.disabled

```diff
@@ -15,7 +15,7 @@
 -- limitations under the License.
 
 set hive.exec.dynamic.partition=true;
-set hive.stats.autogether=false;
+set hive.stats.autogather=false;
```

Comment:
lol!

Reply:
Oh, you weren't telling me to drop this. Sigh. (Restored)

Yes, typos can be funny. But figuring out if they're intended is hard. (Earlier when I was looking at this comment, I parsed it at laughing at an incorrect correction!)

Reply:
not at all :) `hive.stats.autgether` is a non-existent feature toggle :)
but this test seems to be disabled - so its kinda in the gray zone